integration_testing/features/coach/coach-create-quizzes.feature

```diff
@@ -1,50 +1,41 @@
-Feature: Coach create quizzes
+Feature: Coach creates quizzes
   Coach needs to be able to create quizzes from existing content
 
   Background:
     Given I am signed in to Kolibri as a coach user
-    And I am on the *Coach > Quizzes* page
+      And I am on the *Coach > Plan > Quizzes* page
 
   Scenario: Create new quiz
     When I click the *New quiz* button
     Then I see a new *Create new quiz* page
-      And I see empty *Title* and *Number of questions* fields
-      And I see *0 total selected* exercises
+      And I see empty *Title* field, and the default value of 10 in the *Number of questions* field
       And I see a list of channels that contain exercises
-      But I don't see any checkboxes
+        But I don't see any checkboxes
 
   # Given there are no channels that have exercises
   # Then there should not be any channels available to select
   Scenario: Try and fail to create new quiz when no channel on device contains exercises
     When I click the *New quiz* button
     Then I see a new *Create new quiz* page
       And I see empty *Title* and *Number of questions* fields
-      And I see *0 total selected* exercises indicator
       But I don't see any channels
 
```

Comment:
Can probably add a story here for validation of quiz and lesson titles - we have now blocked (case insensitively) the creation of quizzes and lessons within the same class that have the same name (i.e. no quizzes with the same name as each, and no lessons with the same name as each other - you could still have a lesson with the same name as a quiz).

Reply:
Gotcha, will add! 👍